lib/engine/game/g_18_uruguay/step/route.rb

```diff
@@ -12,7 +12,7 @@ def setup
           end
 
           def actions(entity)
-            return %w[].freeze if entity.corporation == @game.rptla
+            return [] if entity.corporation == @game.rptla
```

Comment:
Should this test be `if entity == @game.rptla`?

Reply:
Updated